cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255, null=True)
+    description = models.TextField()
+    duration = models.PositiveIntegerField()
+
+    class Meta:
+        verbose_name_plural = "movies"

```

Comment:
why do u need this ?
![image](https://github.com/mate-academy/py-movie-api/assets/103201010/0da40e74-e253-46a6-8989-08e3e7cbc130)